mlir/include/mlir/Dialect/OpenMP/OpenMPClauseOperands.h

```diff
@@ -251,13 +255,12 @@ using SimdLoopClauseOps =
 using SingleClauseOps = detail::Clauses<AllocateClauseOps, CopyprivateClauseOps,
                                         NowaitClauseOps, PrivateClauseOps>;
 
-// TODO `defaultmap`, `has_device_addr`, `is_device_ptr`, `uses_allocators`
-// clauses.
+// TODO `defaultmap`, `uses_allocators` clauses.
 using TargetClauseOps =
     detail::Clauses<AllocateClauseOps, DependClauseOps, DeviceClauseOps,
                     IfClauseOps, InReductionClauseOps, MapClauseOps,
                     NowaitClauseOps, PrivateClauseOps, ReductionClauseOps,
-                    ThreadLimitClauseOps>;
+                    TargetDeviceClauseOps, ThreadLimitClauseOps>;
```

Comment:
Add here `IsDevicePtrClauseOps` and `HasDeviceAddrClauseOps` instead of `TargetDeviceClauseOps` as well.

Reply:
Done!